terraform/dynamodb.tf

```diff
@@ -0,0 +1,32 @@
+resource "aws_dynamodb_table" "coapi-dynamodb-table" {
```

Comment:
Is it possible to use the DDB schema defined in POJO for building the table ? that way we don't have to change this at 2 places every time we update the schema.
https://github.com/Co-Epi/coepi-backend-aws/blob/master/src/main/kotlin/org/coepi/api/dao/ReportItem.kt

AmazonDynamoDB dynamoDBClient = new AmazonDynamoDBClient();
DynamoDBMapper mapper = new DynamoDBMapper(dynamoDBClient);
CreateTableRequest req = mapper.generateCreateTableRequest(TestClass.class);
// Table provision throughput is still required since it cannot be specified in your POJO
req.setProvisionedThroughput(new ProvisionedThroughput(5L, 5L));
// Fire off the CreateTableRequest using the low-level client
dynamoDBClient.createTable(req);

Reply:
There's no requirement that it has to be created here - it was a manual AWS Console step in the instructions so I automated it here.

That being said, I would like to reuse this TF code to deploy Go and/or Node lambdas, so I have a preference for keeping the code as agnostic as possible.

Reply:
I can create a followup issue.